src/composables/queries/useUserPoolsQuery.ts

```diff
@@ -77,9 +77,7 @@ export default function useUserPoolsQuery(
     });
 
     for (let i = 0; i < pools.length; i++) {
-      const isComposableStableLikePool = isComposableStableLike(
-        pools[i].poolType
-      );
+      const isComposableStableLikePool = isDeep(pools[i]);
```

Comment:
I don't think const is needed anymore, was added to simplify code but function call is clear enough. Just change if statement blow to:
```ts
if (isDeep(pools[i]) {
```